BambooTracker/BambooTracker.pro

```diff
@@ -78,6 +80,11 @@ else:clang|if(gcc:!intel_icc) {
   message("Compiler is version" $$COMPILER_VERSION)
 
   # Temporary known-error downgrades here
+
+  # WIP Requires a deeper look into the warnings to make Qt5 -> Qt6 transition clean
+  greaterThan(QT_MAJOR_VERSION, 5) {
+    CPP_WARNING_FLAGS += -Wno-error=deprecated-declarations -Wno-error=unused-result
+  }
```

Comment:
I hope we can get rid of these again in the future, possibly without any fancy `#ifdef QT6 … #else … #endif` stuff. :slightly_smiling_face: